coil-svg/src/androidMain/kotlin/coil3/decode/SvgDecoder.android.kt

```diff
@@ -20,16 +20,17 @@ import kotlin.math.roundToInt
 import kotlinx.coroutines.runInterruptible
 
 /**
- * A [Decoder] that uses [AndroidSVG](https://bigbadaboom.github.io/androidsvg/) to decode SVG
+ * A [Decoder] that uses [AndroidSVG](https://bigbadaboom.github.io/androidsvg/)
+ * and [SVGDOM](https://api.skia.org/classSkSVGDOM.html/) to decode SVG
```

Comment:
Seems like this should be the docs for the `expect` declaration

Reply:
Done